.github/workflows/junit5-build.yml

```diff
@@ -18,11 +15,11 @@ jobs:
     strategy:
       matrix:
         junit5Minor: [ '5.0', '5.1', '5.2', '6.0', '6.1', '6.2' ]
-    name: with 5.${{ matrix.junit5Minor }}
+    name: Build with JUnit 5.${{ matrix.junit5Minor }}
```

Comment:
i want to start a discussion here, i think this `build` is redundant as it is part of the name of the action, and i think it has a nice reading in the gui without it. I just want to check in what you think about - but that was my initial thought when i removed it.

![image](https://user-images.githubusercontent.com/9987394/89617236-22411b00-d88a-11ea-81bd-0e992109a61c.png)
![image](https://user-images.githubusercontent.com/9987394/89617262-3127cd80-d88a-11ea-8975-3863f38a2d49.png)

Reply:
Yeah, totally agree! Saw it in the UI after the PR. So yes, I will remove that redundancy.

Reply:
i also thought about to rename it like, build with Java 8 on ubuntu - because i think naming wise it is nicer to have the first information aligned rather than the last one

Reply:
maybe we should also name the gradle wrapper validation gha as `validate` and the job `Gradle Wrapper` - so we could theoretically validate more in there, and would also have the same naming scheme

Reply:
Removed the redundancy and now included the OS everywhere, I think this plays well with the workflow name.

Not sure about your last comment. Usually, I'm in favor of validating (in the sense of `mvn validate`) as soon as possible in the main build pipeline, so that the build aborts early in case of a failure.

However, I'm open in this regard. What do you think?

Reply:
what i meant is that the gradle validation could also be a just a file called `validation.yml` instead of `gradle-wrapper-validation.yml` - the name of the GHA could be `Validate` and the name of the (currently only) job could be `gradle wrapper` compared to now where the GHA is named `Gradle Wrapper Validation` and the job is named `validate`. to be consistent with the other namings, and to allow us to add other validations into this GHA :)